ditto/src/main/java/eu/xenit/testing/ditto/internal/DefaultNodeView.java

```diff
@@ -44,18 +51,19 @@ private void processRecursive(RecordLogEntry<Transaction> record) {
     }
 
     private void process(Transaction txn) {
-        log.debug("Replaying {} - with {} writes and {} deletes", txn, txn.getUpdated().size(), txn.getDeleted().size());
+        log.debug("Replaying {} - with {} writes and {} deletes", txn, txn.getUpdated().size(),
+                txn.getDeleted().size());
 
-        txn.getUpdated().forEach(this::add);
-        txn.getDeleted().forEach(this::delete);
+        txn.getUpdated().forEach(this::_add);
```

Comment:
`_` prefix for private methods?

Reply:
The internals and the public methods are very distinctive API's, but it's not very consistently applied. Probably should undo this